### ISSUE TITLE
Video not Playing for me

### ISSUE THREAD

**Issue by Andulien** [MrKrabat/plugin.video.wakanim#15]
Hello, 
so I just setup Wakanim with Kodi. I am able to login and see my lists. However when trying to play an episode (expl. Newest Once Piece episode) it either does not start, or I get an error message, telling me to check the log. Here the log part. I hope this helps to get this fixed as I really would like to use the addon. Thank you :)

PS: Windows 7-x64; Kodi 17.6

```
22:18:41.650 T:5452   ERROR: CImageLoader::DoWork - Direct texture file loading failed for resource://resource.images.studios.white/
22:18:43.814 T:796  NOTICE: VideoPlayer: Opening: https://wmvideocdnstd-wakatest-euno.streaming.media.azure.net/1e9ed391-23c0-436b-830c-5e8cd8a3418a/SAO3_08_DE_HD.ism/Manifest(format=mpd-time-csf,audio-only=false)
22:18:43.814 T:796 WARNING: CDVDMessageQueue(player)::Put MSGQ_NOT_INITIALIZED
22:18:43.814 T:796   ERROR: DXVA::CProcessorHD::IsFormatSupported: Unsupported format 104 for 1.
22:18:43.814 T:796   ERROR: DXVA::CProcessorHD::IsFormatSupported: Unsupported format 105 for 1.
22:18:43.815 T:4740  NOTICE: Creating InputStream
22:18:44.960 T:4740   ERROR: CCurlFile::FillBuffer - Failed: HTTP returned error 400
22:18:44.960 T:4740   ERROR: CCurlFile::Open failed with code 400 for https://wakatest.keydelivery.northeurope.media.azure.net/Widevine/?kid=6a4cd856-0d8f-4c7b-b840-dd3201c2f9b3|Authorization=Bearer%20eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1cm46bWljcm9zb2Z0OmF6dXJlOm1lZGlhc2VydmljZXM6Y29udGVudGtleWlkZW50aWZpZXIiOiI2YTRjZDg1Ni0wZDhmLTRjN2ItYjg0MC1kZDMyMDFjMmY5YjMiLCJDS1R5cGUiOiJDRU5DIiwiaXNzIjoiV2FrYW5pbS5UViIsImF1ZCI6Imh0dHA6Ly93d3cud2FrYW5pbS50diIsImV4cCI6MTU0MzQzNzQ5NywibmJmIjoxNTQzMzUzMTk3fQ.Yw7mueCynvsjHkgrVBuE3AXBGkkrP_C7WyJqPRQtWDU&Content-Type=text%2fxml&Expect&SOAPAction=http%3a%2f%2fschemas.microsoft.com%2fDRM%2f2007%2f03%2fprotocols%2fAcquireLicense&User-Agent=Mozilla%2f5.0%20(Windows%20NT%2010.0%3b%20Win64%3b%20x64)%20AppleWebKit%2f537.36%20(KHTML%2c%20like%20Gecko)%20Chrome%2f60.0.3112.113%20Safari%2f537.36&acceptencoding=gzip%2c%20deflate&postdata=CAESwwsKigsIARLwCQqzAggCEhVDaHJvbWVDRE0tV2luZG93cy14ODYY3qyOjgUijgIwggEKAoIBAQChp2SyN%2foe3bMG20SLvKfCIPL6MwtH9cTL1MvDBlSohXnK1NfQqZLaAK%2bC5R%2fer0UtwTx8QQpd47XkLND594UjxhPIeOE9LaU1V0xe3O0Km5zuqa35aKYH9mT3lp2DMcd8PF1K0cqVwwIk%2bzp1lcujDUfYCuq3TAJmt7boWqkSkGSHAVvDdtmzoS7hg21clsww8Qa5TMJ8NF6o7sfNntqyzHW%2bAlWT8jmKUxEgZpykDH7pxodGCUl%2fFg1buEIrXRK8zwfNSrWOHeCSE5hSPDs1NlLZiuCXCXZ6M04E00lZBmwke3kbBaysQNo4lKGdx0Pa1V%2f6j5CSygiEa0uzjP8JAgMBAAEo%2fiESgAI40qYd4AX%2fArDuIrtIIFeBbHIG6qb8XscIw1HL1vdK2fiEvOIeBa2gvtknr0nZm1sPVdDB56Nm2P4bzsTkZyFSAe14fHRAfRLJW6r%2bNlgUQ%2fAEF53AOuVoNrkLYtiioFwa2aqZNkKEWRp3o4XgeS%2b%2fEAwbOXctHP1o2aC2N3zRCFwF%2b86otvF7tvgbtNpKRv0XdI%2b7AUrIrd8HaQZHXzT%2fprXUc4jpc6N05VOAGrlzsEmnUnFxA05EAL6RlJqNjrortn1h%2bDbg7YFONztGgXhoZIJNyl4VNHIoyzMIbzRplezkphqA5Ddm5dKU0Q0L1pJ3dNaorWc8u4HzxmxsKu1wGrQFCq4CCAESEIDWitLv5OFYkD1F%2b8suiocYrqWOjgUijgIwggEKAoIBAQDr2Qct5A12DjupTitPCl2ce2ijnjz9sYDYgzTY4ejtrp6WDLxTYTwYP5aekFESVeFpGQiR2BmGDADgcbetad6M%2bcRzz2rv3IccTSbGaWncFQf3mRLn0d00%2bvASDsG9GONYR0vJiOWpmArhoLvz2cIrDxjFXiS70%2bJ7HeTOGXSdcwvvlboT5mmlR4MetggVNQHGcJ25C8oeAcXus9gfm6fkLpivbiizNLjaRQ0cPl4TWBCooHBuumuM0by6zWex7KU%2b%2fCv9EsBgLy31FDL3Bt7B5OksAbd%2bJX6O5LhbXz68Iij3kfNWPDtQH3tmqHilvZzoTuJGacuNeWaAKRALNMrnAgMBAAEo%2fiESgAMAsqBvovens6qh7r61e%2b8brvjPVzMPxvaJbt9ookbyYC8WQXwt%2fLiRzcfgodSrc0JsTriiEVKInmb64rXIGLzO1BP%2fv%2b64q55T3%2fl0qPliCQt5rRMQeuLbZDbu2VcFwvtvOKskwAkcU8SjXvHjF%2bZoMf28Q43KVyOYf80PAa8OGF94dp0eoWuhFqE%2bScAECiWsPAcf65YVR5IyFm9AD2svAEYbirbZTpW%2baIEpmZVCsZJSkbNrr9lSZxdYMa0ZyVuqzA6v%2fAq9srftUtZFLWpcK%2biKRMbWfPhKoJEc%2bUf2vSkMdek54ELVqW0O0cGkFFf7KL0Kkj2NeIYOPk9kVWpMspbyj7Y%2fyJZhHVbXPIns%2faSiGRqeS3iTEA%2bq91Ti16twCqRdlfz8hmXlGb5UuR8iY26pUl336qi44swYfbdPB64Zy9DMfih38izi2EtxWm%2b8h0DjhnRlF7CngPLoztSsVR0PuiKA1%2bG6Nv6sYCUNM%2bsjNDojBTEaJiv9BCp8zgkaGwoRYXJjaGl0ZWN0dXJlX25hbWUSBng4Ni0zMhoWCgxjb21wYW55X25hbWUSBkdvb2dsZRoXCgptb2RlbF9uYW1lEglDaHJvbWVDRE0aGAoNcGxhdGZvcm1fbmFtZRIHV2luZG93cxohChR3aWRldmluZV9jZG1fdmVyc2lvbhIJMS40LjguOTAzMggIARAAGAEgARIqCigKEhIQakzYVg2PTHu4QN0yAcL5sxABGhCP%2fIjUvEIXVJCE4B%2blpYQzGAEgtOn23wUwFRqAAoq2RMG0bB86%2bKFryaNGBkJakZCcqLqBHHqYZY0uzdU1DWRsk9rYK1L6Aygr%2bPjYlksxEk6tZIcFULJXUmXtAA%2f5CqrmzyhptyFQ4Bq07mfnDfzIFqJS3zzCYon2vbbyOcmdHwmhPPCV0I3%2bexuEDr4cEZ62V4BSwZ%2bDCAGSWJVGr0fqissE2yQtNB%2b8eRE2mokR659bIN%2fXRG%2bcO1LtPoXUrduT5q1FvzzgG4OBik4wPjZMQoPcNB78wif5WaS%2faF6LxyZOROqeH6YDOPcqMBVk1MuZOQRXgN5Rwr5Fsl1xYH4fjy9kJyVN5jt733YmCjzEiaJb6MFIMUBbrb6CTtg%3d&seekable=0
22:18:44.960 T:4740   ERROR: AddOnLog: InputStream Adaptive: License server returned failure
22:18:44.961 T:4740   ERROR: AddOnLog: InputStream Adaptive: License update not successful (no keys)
22:18:44.961 T:4740   ERROR: AddOnLog: InputStream Adaptive: Initialize failed (SingleSampleDecrypter)
22:18:44.963 T:4740   ERROR: CVideoPlayer::OpenInputStream - error opening [https://wmvideocdnstd-wakatest-euno.streaming.media.azure.net/1e9ed391-23c0-436b-830c-5e8cd8a3418a/SAO3_08_DE_HD.ism/Manifest(format=mpd-time-csf,audio-only=false)]
22:18:44.963 T:4740  NOTICE: CVideoPlayer::OnExit()
22:18:44.968 T:796   ERROR: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.wakanim/?fanart=https%3A%2F%2Fcdn1.wakanim.tv%2Fmovies-pictures%2F3e80ef9e-b8d8-4d93-a730-b75bed98d091.jpg&mode=videoplay&title=SWORD+ART+ONLINE+-Alicization-+%28OmU%29+Folge+8&url=%2Fde%2Fv2%2Fcatalogue%2Fepisode%2F7315%2Fsword-art-online-alicization-omu-arc-1-folge-8-omu&progress=0&playcount=0&plot=Arc+1&thumb=https%3A%2F%2Fcdn1.wakanim.tv%2Fmovies-pictures%2F3e80ef9e-b8d8-4d93-a730-b75bed98d091.jpg]
22:18:44.971 T:796  NOTICE: CVideoPlayer::CloseFile()
22:18:44.971 T:796  NOTICE: VideoPlayer: waiting for threads to exit
22:18:44.971 T:796  NOTICE: VideoPlayer: finished waiting
22:18:44.971 T:796  NOTICE: CVideoPlayer::CloseFile()
22:18:44.971 T:796  NOTICE: VideoPlayer: waiting for threads to exit
22:18:44.971 T:796  NOTICE: VideoPlayer: finished waiting
22:18:55.063 T:796   ERROR: DXGI_ERROR_INVALID_CALL

```

**Comment by Andulien** [comment]
Some further Input:
- Other Addons work with the InputStream widevine, so I doubt it is because of that.
- I currently am using the 14 day premium trial, could this be a reason why?
- Videoplayback works fine in Chrome, however I had to approve the new device first. However since this is device bound, I guess this should not have an affect on whether I use Chrome or Kodi right?

Let me know if there is anything else I can provide you with. I also updated a bit the log to include more data.
